layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,227 @@
+      <li>Alton Sterling</li>
+      <li>Anthony Hill</li>
+      <li>Anthony Lamar Smith</li>
+      <li>Antwon Rose II</li>
+      <li>Ariane McCree</li>
+      <li>Botham Jean</li>
+      <li>Brandon Webber</li>
+      <li>Brendon Glenn</li>
+      <li>Breonna Taylor</li>
+      <li>Charly Keunang</li>
+      <li>Christian Taylor</li>
+      <li>Christopher Whitfield</li>
+      <li>Danny Ray Thomas</li>
+      <li>Dante Parker</li>
+      <li>Darius Tarver</li>
+      <li>DeAndre Ballard</li>
+      <li>DeJuan Guillory</li>
+      <li>Delrawn Small</li>
+      <li>De’Von Bailey</li>
+      <li>Dontre Hamilton</li>
+      <li>D’ettrick Griffin</li>
+      <li>Emantic Fitzgerald Bradford Jr</li>
+      <li>Eric Garner</li>
+      <li>Eric Logan</li>
+      <li>Ezell Ford</li>
+      <li>Finan H. Berhe</li>
+      <li>Freddie Gray</li>
+      <li>George Floyd</li>
+      <li>Gregory Gunn</li>
+      <li>Gregory Hill Jr</li>
+      <li>JaQuavion Slaton</li>
+      <li>Jamar Clark</li>
+      <li>Jamarion Robinson</li>
+      <li>Jamee Johnson</li>
+      <li>Jemel Roberson</li>
+      <li>Jerame Reid</li>
+      <li>Jimmy Atchison</li>
+      <li>John Crawford III</li>
+      <li>Jonathan Ferrell</li>
+      <li>Jonathan Hart</li>
+      <li>Jordan Baker</li>
+      <li>Jordan Edwards</li>
+      <li>Julius Johnson</li>
+      <li>Kajieme Powell</li>
+      <li>Keith Childress</li>
+      <li>Keith Lamont Scott</li>
+      <li>Kendrec McDade</li>
+      <li>Kwame Jones</li>
+      <li>Laquan McDonald</li>
+      <li>Larry Jackson Jr</li>
+      <li>Manuel Loggins Jr</li>
+      <li>Mario Woods</li>
+      <li>Maurice Granton</li>
+      <li>Michael Brown</li>
+      <li>Michael Dean</li>
+      <li>Mike Ramos</li>
+      <li>Miles Hall</li>
+      <li>Natosha McDade</li>
+      <li>Patrick Harmon</li>
+      <li>Philando Castile</li>
+      <li>Quintonio LeGrier</li>
+      <li>Ramarley Graham</li>
+      <li>Robert Lawrence White</li>
+      <li>Rumain Brisbon</li>
+      <li>Ryan Twyman</li>
+      <li>Samuel David Mallard</li>
+      <li>Samuel DuBose</li>
+      <li>Sandra Bland</li>
+      <li>Sean Reed</li>
+      <li>Stephon Clark</li>
+      <li>Steven Demarco Taylor</li>
+      <li>Tamir Rice</li>
+      <li>Terence Crutcher</li>
+      <li>Terrance Franklin</li>
+      <li>Terrence Sterling</li>
+      <li>Tony McDade</li>
+      <li>Tony Robinson</li>
+      <li>Trayvon Martin</li>
+      <li>Victor White III</li>
+      <li>Walter Scott</li>
+      <li>Wendell Allen</li>
+      <li>William Green</li>
+      <li>Willie McCoy</li>
+      <li>Yassin Mohamed</li>
+      <li>... <a href="https://mappingpoliceviolence.org/unarmed">and</a> <a href="https://mappingpoliceviolence.org/unarmed2014">countless</a> <a href="https://apps.npr.org/documents/document.html?id=6933593-NPR-CodeSwitch-Saytheirnameslistv3">more</a> <a href="https://ebwiki.org/">names</a> not listed here whose lives were taken by violence and brutality.</li>
+    </ul>
```

Comment:
```suggestion
    </ul>
    <p>We've listed the names of the Black folks who have been murdered by police brutality in the U.S. dating back to 2009, both because the U.S. is the origin of the Black Lives Matter movement and currently the heart of the resistance.</p>

<p>With that said, we fully recognize that white supremacy and police brutality is a global problem. While we have not listed all of Black people who have lost their lives to police murder globally here. Their lives mattered.</p>
```

Note that I've removed the additional sentence about a global list - I more than welcome one, but in my own searching I've been unable to find one.

A sincere thank you to @tatianamac for help on this 🙏

Reply:
(I don't know how to make a suggestion of a suggestion here [and thank you for your patience; I should have used suggesting mode to begin with but was on the phone], but can we make the last sentence syntactically correct?)

hile we have not listed all of Black people who have lost their lives to police murder globally here, their lives mattered

Reply:
@tatianamac fixed 👍